controller_joint_names: ['', 'rear_wheel_left_joint', 'rear_wheel_right_joint', 'front_wheel_left_steering_joint', 'front_wheel_left_joint', 'front_wheel_right_steering_joint', 'front_wheel_right_joint', ]
# Robot name goes here
Robot_Assembly: 
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  front_wheel_left_steering_controller:
    type: effort_controllers/JointPositionController
    joint: "front_wheel_left_steering_joint"  
    pid : {p: 100.0, i: 0.02, d: 4.0} 

  front_wheel_right_steering_controller:
    type: effort_controllers/JointPositionController
    joint: "front_wheel_right_steering_joint"  
    pid : {p: 100.0, i: 0.02, d: 4.0}

  front_wheel_left_controller:
    type: velocity_controllers/JointVelocityController 
    joint: "front_wheel_left_joint"  
    pid : {p: 100.0, i: 2, d: 10.0}

  front_wheel_right_controller:
    type: velocity_controllers/JointVelocityController 
    joint: "front_wheel_right_joint"  
    pid : {p: 100.0, i: 2, d: 10.0}

  rear_wheel_left_controller:
    type: velocity_controllers/JointVelocityController 
    joint: "rear_wheel_left_joint"  
    pid : {p: 100.0, i: 0.01, d: 10.0}

  rear_wheel_right_controller:
    type: velocity_controllers/JointVelocityController 
    joint: "rear_wheel_right_joint"  
    pid : {p: 100.0, i: 0.01, d: 10.0}  

 